tunder/lib/src/database/operations/contracts/delete_operation.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:tunder/database.dart';
+import 'package:tunder/_common.dart';
+import 'package:tunder/src/database/operations/postgres/postgres_delete_operation.dart';
+
+abstract class DeleteOperation {
+  factory DeleteOperation.forDatabase(Symbol driver) {
```

Comment:
I'd either rename the factory constructor to "forDriver" or rename the Symbol parameter to "database" to improve the naming consistency.